src/main/webapp/assets/modules/technical-detail/technical-detail.con.js

```diff
@@ -1,3 +1,17 @@
-techlooper.controller("technicalDetailController", function() {
+techlooper.controller("technicalDetailController", function (utils, connectionFactory, $routeParams,
+                                                             technicalDetailService, $scope, $timeout, jsonValue) {
```

Comment:
Line is too long.<br>'jsonValue' is defined but never used.<br>'$timeout' is defined but never used.